### PR TITLE
Add azure pipeline

### PR DESCRIPTION
**Pull request by arseniy-sonar**

### CODE REVIEW

File: README.adoc [hunk 1]
```diff
@@ -1,3 +1,46 @@
+= C++ example project scanned on SonarCloud using Azure Pipelines
+// URIs:
+:uri-qg-status: https://sonarcloud.io/dashboard?id=sc-sq-c-family-examples_windows-cmake-azure-sc
+:img-qg-status: https://sonarcloud.io/api/project_badges/measure?project=sc-sq-c-family-examples_windows-cmake-azure-sc&metric=alert_status
+:uri-build-status: https://dev.azure.com/sc-sq-c-family-examples/windows-cmake-azure-sc/_build/latest?definitionId=2&branchName=azure-sc-analysis
```
Comment (tomasz-kaminski-sonarsource) [inline README.adoc]: Remove the default branch.

File: azure-pipelines.yml [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+  variables:
+    BUILD_WRAPPER_DOWNLOAD_URL: https://sonarcloud.io/static/cpp/build-wrapper-win-x86.zip
+  steps:
+  - task: SonarCloudPrepare@1
+    inputs:
+      SonarCloud: 'SonarCloud' # Name of the SonarCloud service connection you created
+      organization: 'sc-sq-c-family-examples'
+      scannerMode: 'CLI'
+      configMode: 'file'
+  - bash: |
```
Comment (tomasz-kaminski-sonarsource) [inline azure-pipelines.yml]: Nice !

**Review by tomasz-kaminski-sonarsource — APPROVED** [review]
Just remove the branch name and we are good to go.

File: azure-pipelines.yml (hunk 2 above)
Comment (tomasz-kaminski-sonarsource) [reply azure-pipelines.yml]: After a bit of consideration. Do you think that using bash in the examples for Windows is helpful? I mean if people are building on windows, the bash syntax may be alien to them. Maybe it be worthwhile to migrate that to powershell, or something else windows native.

Comment (arseniy-sonar) [reply azure-pipelines.yml]: Yep, rewritten in Powershell

File: README.adoc (hunk 1 above)
Comment (arseniy-sonar) [reply README.adoc]: done.

**Pull request merged**